### ISSUE TITLE
Hit Delay?

### ISSUE THREAD

**Issue by Owen145** [Shampaggon/CrackShot#74]
I saw on a server this person made their guns instantly deal damage with no hit delay. Every bullet connected and did damage, minecraft has a default hit delay thing and I was wondering if crackshot made a way to get around this?

**Comment by AlexSerb-RU** [comment]
> Я видел на сервере, что этот человек сделал свое оружие мгновенно наносить урон без задержки попадания. Каждая пуля соединялась и наносила урон, у Майнкрафта есть задержка попадания по умолчанию, и мне было интересно, есть ли у crackshot способ обойти это?

Reset_Hit_Cooldown:
This is a particularly useful ability. You may have realized by now that hitting an entity will grant them invulnerability for a split second. This prevents them from getting hit too many times in one second. Setting this to true will reset the temporary invulnerability. If this node is false, weapons with a high rate of fire, or weapons like shotguns that fire more than one projectile, would not be as effective.

In a nutshell: This ability is a must-have for most firearms, especially for shotguns, which rely on having more than one projectile hit the enemy at the same time.
